addons/mail/static/src/models/thread/thread.js

```diff
@@ -416,6 +416,23 @@ function factory(dependencies) {
             this.update({ areAttachmentsLoaded: true });
         }
 
+        async fetchUpdateSuggestedPartners() {
+            const result = await this.async(() => this.env.services.rpc({
+                route: '/mail/get_suggested_recipients',
+                params: {
+                    model: this.model,
+                    res_ids: [this.id],
+                },
+            }));
+            if (result[this.id]) {
+                this.update({
+                    suggestedPartners: [['insert-and-replace', result[this.id].map(data => {
+                        return this.env.models['mail.suggested_partner'].convertData(data);
+                    })]],
+                });
+            }
+        }
```

Comment:
Not fond in the way it's used. Too much imperative, so very fragile to code changes.
Using `_updateAfter` is deprecated (we eventually want to remove/replace them)... however, using this is much more preferred than current solution (much more declarative, so more robust)